auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+                return True, {
+                    'access': str(new_access),
+                    'refresh': str(new_refresh),
+                    'token_type': 'Bearer',
+                    'expires_in': int(
+                        getattr(settings, 'SIMPLE_JWT', {}).get(
+                            'ACCESS_TOKEN_LIFETIME',
+                            timedelta(minutes=5)
+                        ).total_seconds()
+                    )
+                }
+            else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)